mlflow/utils/search_utils.py

```diff
@@ -456,14 +457,22 @@ def _validate_order_by_and_generate_token(cls, order_by):
                 error_code=INVALID_PARAMETER_VALUE,
             )
         statement = parsed[0]
+        ttype_for_timestamp = (
+            TokenType.Name.Builtin
+            if Version(sqlparse.__version__) >= Version("0.4.3")
+            else TokenType.keyword
```

Comment:
```suggestion
            else TokenType.Keyword
```

to include https://github.com/mlflow/mlflow/pull/6901

Reply:
Awesome! Included!